op-batcher/batcher/config.go

```diff
@@ -88,17 +88,16 @@ type CLIConfig struct {
 	Stopped bool
 
 	TxMgrConfig      txmgr.CLIConfig
-	RPCConfig        rpc.CLIConfig
 	LogConfig        oplog.CLIConfig
 	MetricsConfig    opmetrics.CLIConfig
 	PprofConfig      oppprof.CLIConfig
 	CompressorConfig compressor.CLIConfig
+	RPCFlag          oprpc.CLIConfig
 }
 
 func (c CLIConfig) Check() error {
-	if err := c.RPCConfig.Check(); err != nil {
-		return err
-	}
+	// TODO: check the sanity of flags loaded directly like MaxPendingTransactions
```

Comment:
I noticed that currently we don't check any of flags that are loaded directly. I didn't wanted to increase the scope of this PR, but still wanted to catch this issue so I created a TODO. Let me know if I should delete this.

Reply:
Can you also create a github issue & include the issue number next to the todo comment?

Reply:
done

Issue: https://github.com/ethereum-optimism/optimism/issues/7512

Diff: https://github.com/ethereum-optimism/optimism/pull/7482/commits/1bc388eca88d5b5e69622643bbfc45d86c8bea6a